prowler/providers/aws/services/dms/dms_service.py

```diff
@@ -11,7 +11,9 @@ def __init__(self, provider):
         # Call AWSService's __init__
         super().__init__(__class__.__name__, provider)
         self.instances = []
+        self.endpoints = {}
         self.__threading_call__(self.__describe_replication_instances__)
+        self.__threading_call__(self.__describe_endpoints__)
```

Comment:
```suggestion
        self.__threading_call__(self._describe_replication_instances)
        self.__threading_call__(self._describe_endpoints)
```
Please, add testing for the new method in `dms_service_test.py`